ePBFT/src/pbft/libbyz/Replica.cpp

```diff
@@ -2878,7 +2882,7 @@ void Replica::vtimer_handler(void* owner)
 {
   PBFT_ASSERT(replica, "replica is not initialized\n");
 
-  if (!replica->delay_vc())
+  if (!replica->delay_vc() && replica->f() > 0)
```

Comment:
Is there a specific reason that this here is needed? I am curious as to what behaviour are you seeing. View changes should ideally never need to be triggered since requests are processed right away when `f == 0`

Reply:
Discussed offline